packages/strapi-design-system/src/DateTimePicker/__tests__/__snapshots__/DateTimePicker.spec.js.snap

```diff
@@ -402,7 +402,7 @@ exports[`DateTimePicker snapshots the component 1`] = `
                     class="c17"
                     id="timepicker-5-content"
                   >
-                    12:00
+                    10:00
```

Comment:
Just wondering: why does this need an update suddenly? Is this test flaky?

Reply:
I think the problem was my TMZ config